core/handle_execute_client.py

```diff
@@ -18,6 +18,8 @@ def handle_execute_client(session: Session, params: Any) -> None:
 
     if command_name == "metals-goto-location":
         goto_location(session, args)
+    elif command_name == 'metals-diagnostics-focus':
+        sublime.run_command('lsp_goto_diagnostic')
```

Comment:
`sublime.run_command` runs `ApplicationCommand` but `LspGotoDiagnostic` is a `WindowsCommand`. Does that actually work?